### DISCUSSION TITLE
MacOS installation instruction is imcomplete

### DISCUSSION THREAD

**Discussion by ShivamJoker** [ThioJoe/YT-Spammer-Purge#542] [General]
The requirements.txt isn't correct, you will have to install tk via brew to make the app run

```sh
brew install python-tk
```

**Comment by Dobby233Liu** [comment]
I think you only need to install that if you install python through homebrew, python.org builds come with tk, right?

**Reply by ShivamJoker** [reply]
I don't know about that, but this should be in the wiki page at least.
People like me uses brew for everything, installing from GUI sucks
